src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+.formField {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.personal-Information,
+.registration,
+.interestingFacts,
+.addInfo {
+  margin-bottom: 20px;
+  padding-bottom: 0;
+}
```

Comment:
why not just give all of them a single classname with more generic name? Seems you don't use most of them and they are all covered with a couple of css-rules. Let's simplify this a little